templates/admin/bulkReplace.tpl

```diff
@@ -3,33 +3,61 @@
 {block "title"}Înlocuire în masă{/block}
 
 {block "content"}
-  <h3>Înlocuire în masă: {$searchResults|count} de definiții</h3>
+  <h3>Înlocuire în masă</h3>
 
   <div class="panel-admin">
       <div class="panel panel-default">
-        <div class="panel-heading">
+        <div class="panel-heading" id="panel-heading">
           <i class="glyphicon glyphicon-user"></i>
             {$modUser}
+          <div class="checkbox-inline">
+              <input type="checkbox" name="checkbox-all" id="checkbox-all" />
+          </div>
         </div>
-
-      {foreach $diffs as $d}
-      {if isset($d)}
-        <div class="panel-body">
-          <p>{$d}</p>
+        <div class="panel-body" id="panel-body">
+            {foreach $searchResults as $row}
+              <div class="checkbox-inline" id="checkbox">
+                  <input type="checkbox" name="checkbox" id="{$row->definition->id}" value="{$row->definition->id}" />
```

Comment:
_checkbox_ e un nume cam generic. O poți numi altcumva, _whitelist_ sau _includedIds_ sau cum vrei tu?

Reply:
„name”-ul div-ului sau „id”-ul input-ului?

Reply:
Da, la `name` mă refeream. Hm, acum că-mi spui, și ID-ul strict numeric este un pic periculos (dacă facem la fel și cu alte obiecte și apar duplicate)? Știu două metode să eviți asta: îi dai un prefix ca `include_{$row->definition->id}` sau pui ID-ul într-un atribut HTML5, `data-definitionId` sau altcumva.

Reply:
Da, puțin leneș aici. N-am vrut să mă complic cu prefix. Modific.